src/routes/(public)/(guest)/login/+page.svelte

```diff
@@ -129,3 +131,9 @@
         </li>
     </svelte:fragment>
 </Unauthenticated>
+
+<style>
+    .auth-provider-wrapper {
+        margin-bottom: var(--gap-s, 8px);
+    }
```

Comment:
we can inline on the div itself with svelte syntax since its only one property like `style:margin-bottom='...'`